share/locale/tours_fr.ts

```diff
@@ -0,0 +1,344 @@
+    </message>
+    <message>
+        <source>Welcome to MuseScore! This brief tour will guide you through the MuseScore user interface.</source>
+        <comment>welcome</comment>
+        <translation>Bienvenue dans MuseScore ! Cette brève visite vous guidera à travers l&apos;interface graphique de MuseScore.</translation>
+    </message>
+    <message>
+        <source>The Playback Controls toolbar allows you to play/pause your score and control various aspects of playback.</source>
+        <comment>welcome</comment>
+        <translation>La barre d&apos;outils des commandes de Lecture vous permet de Lire / Mettre en pause votre partition et contrôle différents aspects de la lecture.</translation>
+    </message>
+    <message>
+        <source>The Note Input toolbar allows you to enter notes and rests into your score.</source>
+        <comment>welcome</comment>
+        <translation>La barre d&apos;outils de Saisie de notes vous permet d&apos;ajouter des notes et des silences dans votre partition.</translation>
+    </message>
+    <message>
+        <source>The Palettes allow you to add most other notation elements to your score.</source>
+        <comment>welcome</comment>
+        <translation>Les Palettes vous permettent d&apos;ajouter la majorité des autres éléments de notation à votre partition.</translation>
+    </message>
+    <message>
+        <source>Each palette contains items that can be added to the score.</source>
+        <comment>show-palette</comment>
+        <translation>Chaque palette contient des éléments pouvant être ajoutés à la partition.</translation>
+    </message>
+    <message>
+        <source>Welcome to the Palettes, where you will add various elements to your score. Click &quot;Next&quot; to learn more.</source>
+        <comment>show-palette</comment>
+        <translation>Voici les Palettes, où vous pouvez ajouter différents éléments à votre partition. Appuyez sur &quot;Suivant&quot; pour en apprendre plus.</translation>
+    </message>
+    <message>
+        <source>To enter a note of the selected duration, type its letter name or click on the staff.
+    Press Ctrl+Up/Down (macOS: Cmd+Up/Down) to change octave.</source>
+        <comment>note-input</comment>
+        <translation>Pour ajouter une note de la durée sélectionnée, appuyez sur sa lettre ou cliquez sur la portée.
+Appuyez sur Ctrl + Haut / Bas (macOS : Cmd + Haut / Bas) pour changer d&apos;octave.</translation>
+    </message>
+    <message>
+        <source>To add an accidental to a note, press the shortcut Up or Down to raise or lower its pitch,
+    or click one of these icons.</source>
+        <comment>note-input</comment>
+        <translation>Pour ajouter une altération accidentelle à une note, appuyez sur les flèches Haut ou Bas pour monter ou descendre la note d&apos;un demi-ton, 
+ou cliquez sur l&apos;un de icônes appropriés.</translation>
```

Comment:
Should not it be "cliquez sur l'un**e** de**s** icône**s** approprié**e**s"

Reply:
all these need to get fixed/changed on Transifex. This PR is just to have the ts file in the repo and a qm file in the package, so updating translations can work on it

Reply:
oh, ok!
if I want to help improve the translation, it is here ?
https://www.transifex.com/musescore/musescore/

Reply:
Yes. I've fixed those 4 strings there meanwhile though